src/test/java/com/keunsori/keunsoriserver/admin/init/ApiTestWithWeeklyScheduleInit.java

```diff
@@ -0,0 +1,30 @@
+package com.keunsori.keunsoriserver.admin.init;
+
+import com.keunsori.keunsoriserver.common.ApiTest;
+import com.keunsori.keunsoriserver.domain.admin.reservation.domain.WeeklySchedule;
+import com.keunsori.keunsoriserver.domain.admin.reservation.repository.WeeklyScheduleRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+
+public class ApiTestWithWeeklyScheduleInit extends ApiTest {
+    @Autowired
+    private WeeklyScheduleRepository weeklyScheduleRepository;
+
+    @BeforeEach
+    void setupWeeklySchedule() {
+        if (weeklyScheduleRepository.count() == 0) {
+            for (DayOfWeek day : DayOfWeek.values()) {
+                WeeklySchedule schedule = WeeklySchedule.builder()
+                        .dayOfWeek(day)
+                        .isActive(false)
+                        .startTime(LocalTime.of(10, 0))
+                        .endTime(LocalTime.of(23, 0))
+                        .build();
+                weeklyScheduleRepository.save(schedule);
+            }
+        }
+    }
```

Comment:
```suggestion
    @Autowired
    private WeeklyScheduleInitializer weeklyScheduleInitializer;

    @BeforeEach
    void setupWeeklySchedule() {
        weeklyScheduleInitializer.initializeSchedules();
    }
```

이부분 이렇게 바꾸면 WeeklyScheduleInitializer에 있는 코드를 그대로 사용하게 되어서
실제 운영 코드로 테스트를 돌릴 수 있고 Initializer 코드 수정시에도 누락의 위험이 사라져서 좋을 것 같습니다!